uwp/Tree-Navigator/Getting-Started.md

```diff
@@ -258,4 +258,4 @@ treeNavigator.Items.Add(New SfTreeNavigatorItem() With {.Header="Silverlight (30
 
 {% endtabs %}
 
-![Creating SfTreeNavigator control](Overview-images/Overview-img1.jpeg)
\ No newline at end of file
+![Creating SfTreeNavigator control](overview-images/uwp-tree-navigator-creating.jpeg)
```

Comment:
creating-tree-navigator-control-in-uwp
change like this.
Image name should be proper ending.